Makefile

```diff
@@ -328,6 +328,10 @@ lint-fix: ##@test Run code style checks and fix issues
 	sh scripts/lint-trailing-newline.sh --fix && \
 	yarn prettier
 
+clean-namespaces: export TARGET := clojure
+clean-namespaces: ##@ Clean up Clojure namespaces (remove unused & sort)
+	clojure-lsp clean-ns
```

Comment:
There is a discrepancy in how `zprint` formats namespaces and `clojure-lsp clean-ns`. That's why running `make lint-fix` won't fix sorting namespaces and that's why I kept this make target separate. @ulisesmac recently mentioned this formatting discrepancy here https://github.com/status-im/status-mobile/pull/17496#discussion_r1355844780

In VS Code, Vim and Emacs I know it's common and simple to clean up namespaces, either automatically after every save or by a keyboard shortcut, so hopefully this means most collaborators don't need to rely on the slow `clojure-lsp clean-ns` call. We could always enable the linter for unsorted namespaces at the warning level too, but then the code would still be merged with unsorted namespaces every now and then, so not much to gain I'd say.